### PR TITLE
D ui3 305 agis implement highlight for individual features e.g. from receive report

### PR DESCRIPTION
**Pull request by KatKatKateryna**
1. Introduce struct holding LayerURI (represented as string because this is how native API is treating LayersURI), featureId, MapMember (actual layer object on the map)
2. Adapt Report building and Highlight functionality to use this instead of flat string with LayerURI
3. Add functionality to select individual features if FeatureID is available

### CODE REVIEW

File: DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/Utils/ObjectID.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+using ArcGIS.Desktop.Mapping;
+
+namespace Speckle.Connectors.ArcGIS.Utils;
+
+public struct ObjectID
```
Comment (oguzhankoral) [inline DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/Utils/ObjectID.cs:5]: I would omit MapMember from this struct, it is confusing why objectID checks for MapMember as below. 

```c#
if (objectId.MapMember == null)
{
   continue;
}
``` 

What I expect from `ObjectID` is just id, if you include MapMember it becomes something else. In this case, we might need another struct that has MapMember too instead considering its nullability on `ObjectID` struct.

**Review by oguzhankoral — CHANGES_REQUESTED** [review]
Overall, I liked the extraction on object id which was always confusing for me, maybe I'm greedy but I believe we have an opportunity to have it more clean way. Then good to go 🚀

**Comment by KatKatKateryna** [comment]
converting to draft to update documentation

**Review by oguzhankoral — APPROVED** [review]
Tested and LGTM 🚀

**Pull request merged**